Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableToggleCollectionInspector.cs

```diff
@@ -31,10 +31,11 @@ public override void OnInspectorGUI()
 
             if (Application.isPlaying && instance != null && GUI.changed)
             {
-                int index = instance.CurrentIndex;
-                if (index >= instance.ToggleList.Length || index < 0)
+                instance.CurrentIndex = Mathf.Clamp(instance.CurrentIndex, 0, instance.ToggleList.Length - 1);
```

Comment:
This is great, if I updated currentIndex in the inspector to like -1, then it won't accept the value. 

On problem, is ToggleList is still exposed. Since it's more unlikley someone will change the ToggleList via inspector at runtime, I'd say it's fine to just disable the GUI for that property. 

You'll have to get rid of `base.OnInSpectorGUI()` and do your own rendering

``
... OnEnable()
{
toggleListProperty = serializedObject.FindProperty("toggleList");
...
}

oninspectorgui()
{

new EditorGUI.DisabledGroupScope(Application.isPlaying))
{
EditorGUILayout.PropertyFIeld(toggleListProperty);
}

EditorGUILayout.PropertyFIeld(currentIndexProperty);

...
```

Reply:
Disabled editing the ToggleList through the inspector during playmode. 
![image](https://user-images.githubusercontent.com/53493796/68520770-5e8a1100-024f-11ea-9b3b-52d072dbfd95.png)